### PR TITLE
Fix OpenAPI specs

### PR DESCRIPTION
**Pull request by provokateurin**
## Summary

Some stuff I forgot to commit and small fixes for union types. A future CI will catch the former problems.

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [ ] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [ ] Screenshots before/after for front-end changes
- [ ] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [ ] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

**Review by bigcat88 — APPROVED** [review]
Does it a good idea to have such large description with an example of the reply?

**Comment by provokateurin** [comment]
Sorry, I don't understand what you mean. Can you rephrase for me?

**Comment by bigcat88** [comment]
I am about the part with the _description: Output can look like this: ..._

**Comment by provokateurin** [comment]
Ah I see, that description isn't on me though. It's just extracted from the comment of the method.

**Review by julien-nc — APPROVED** [review]
Is it a fix that will have to be done on each future generated OpenAPI specs?

**Comment by provokateurin** [comment]
If the openapi-extractor is updated and the output is changed, yes. I will add it as a dev dependency and also setup CI. Then the tool will be upgraded by bumping the dependency and thus needing to regenerate the specs. I expect the updates to get less and less in the future as the output gets better and better.
